.pre-commit-config.yaml

```diff
@@ -154,61 +154,21 @@ repos:
       # NOTE: CI automations run both checks. Whenever the versions are
       # NOTE: updated in this config, a symmetric action must be performed
       # NOTE: in `tox.ini` as well.
-      - &_base-mypy-setup-map
-        id: mypy
+      - id: mypy
         alias: mypy-py310
         name: MyPy, for Python 3.10
-        additional_dependencies:
-          - lxml == 4.6.4 # needed for `--txt-report`
-          - types-docutils
+        additional_dependencies: &_mypy_deps
           - types-PyYAML
+          - types-dataclasses # Needed for checking py3.6 with a later interpreter
```

Comment:
I'd prefer different deps for different envs, I don't think they should influence each other.

Reply:
Ok.